### PR TITLE
Updating build.sbt and README.

### PR DESCRIPTION
**Pull request by jackheslop96**

### CODE REVIEW

File: app/services/package.scala [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import cats.data.NonEmptySet
+
+package object services {
+
+  implicit class RichNonEmptySet[T](value: NonEmptySet[T]) {
+    def toSeq: Seq[T] = value.toNonEmptyList.toList
```
Comment (joecefnhedog) [inline app/services/package.scala:22]: Does this need to be made a Set?

File: it/test/connectors/ReferenceDataConnectorSpec.scala [hunk 2]
```diff
@@ -234,8 +245,8 @@ class ReferenceDataConnectorSpec extends ItSpecBase with ScalaCheckPropertyCheck
   "Reference Data" - {
 
     "getCustomsOfficesForCountry" - {
-      val countryCode = CountryCode("GB")
-      val url         = s"/$baseUrl/filtered-lists/CustomsOffices?data.countryId=${countryCode.code}&data.roles.role=DES"
+      val countryIds = Seq("GB", "XI")
+      val url        = s"/$baseUrl/lists/CustomsOffices?data.countryId=XI&data.countryId=GB&data.roles.role=DES"
```
Comment (joecefnhedog) [inline it/test/connectors/ReferenceDataConnectorSpec.scala:249]: could this dynamically use the countryIds val above

File: it/test/connectors/ReferenceDataConnectorSpec.scala [hunk 3]
```diff
@@ -244,20 +255,24 @@ class ReferenceDataConnectorSpec extends ItSpecBase with ScalaCheckPropertyCheck
...
       "must throw a NoReferenceDataFoundException for an empty response" in {
-        checkNoReferenceDataFoundResponse(url, connector.getCustomsOfficesForCountry(countryCode))
+        val countryId = "AR"
+        val url       = s"/$baseUrl/lists/CustomsOffices?data.countryId=AR&data.roles.role=DES"
+        checkNoReferenceDataFoundResponse(url, connector.getCustomsOfficesForCountry(countryId))
       }
 
       "must return an exception when an error response is returned" in {
-        checkErrorResponse(url, connector.getCustomsOfficesForCountry(countryCode))
+        val countryId = "GB"
```
Comment (joecefnhedog) [inline it/test/connectors/ReferenceDataConnectorSpec.scala:273]: could this use the Constants file

File: it/test/connectors/ReferenceDataConnectorSpec.scala (hunk 2 above)
Comment (joecefnhedog) [reply it/test/connectors/ReferenceDataConnectorSpec.scala:249]: and the constants file

Comment (jackheslop96) [reply it/test/connectors/ReferenceDataConnectorSpec.scala:249]: Not going to make it dynamic as want to be sure that the url is being formed correctly with the query params included

File: it/test/connectors/ReferenceDataConnectorSpec.scala (hunk 3 above)
Comment (jackheslop96) [reply it/test/connectors/ReferenceDataConnectorSpec.scala:273]: Going to leave as is if that's ok. It's largely an arbitrary value

**Review by joecefnhedog — APPROVED** [review]

**Pull request merged**